config/main.py

```diff
@@ -542,32 +586,132 @@ def config():
 config.add_command(nat.nat)
 
 @config.command()
+@click.option('-n', '--namespace', help='Namespace name')
 @click.option('-y', '--yes', is_flag=True, callback=_abort_if_false,
-                expose_value=False, prompt='Existing file will be overwritten, continue?')
-@click.argument('filename', default='/etc/sonic/config_db.json', type=click.Path())
-def save(filename):
+                expose_value=False, prompt='Existing files will be overwritten, continue?')
+@click.argument('filename',nargs=-1)
```

Comment:
keep a space after comma

Reply:
I also suggest for multi-npu case, the filename should be a a list files separated by comma as a single string.

Reply:
Made the change to tahe the argument from user as a single string with files separated by comma.